apps/files_sharing/lib/Controller/ShareAPIController.php

```diff
@@ -786,6 +787,84 @@ public function getShares(
 		return new DataResponse($formatted);
 	}
 
+
+	/**
+	 * The getSharesComlete function.
```

Comment:
```suggestion
	 * The getSharesComplete function.
```

I would prefer something like `getAllShares` but naming is alway a personal thing ;)